src/components/pages/AdminMeetingsDashBoard/AdminMeetingDash.js

```diff
@@ -0,0 +1,159 @@
+import React, { useState, useEffect } from 'react';
+import ViewAllMeetings from './ViewAllMeetings/ViewAllMeetings';
+import useAxiosWithAuth0 from '../../../hooks/useAxiosWithAuth0';
+import { Link } from 'react-router-dom';
+import CreateModal from './CrudModals/CreateModal';
+import DeleteModal from './CrudModals/DeleteModal';
+import UpdateModal from './CrudModals/UpdateModal';
+
+const AdminMeetingDash = () => {
+  const [meetings, setMeetings] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [allMentors, setAllMentors] = useState([]);
+  const [allMentees, setAllMentees] = useState([]);
+  const { axiosWithAuth } = useAxiosWithAuth0();
```

Comment:
minor change required
const axiosWithAuth  = useAxiosWithAuth0();

Reply:
I updated all instances new instances of axios to utilize the updated useAxiosWithAuth0 hook.